tests/test_creation/test_combine_with_reference_feature.py

```diff
@@ -113,7 +113,7 @@ def test_all_binary_operation(df_vartypes):
             "Age_mul_Marks": [18.0, 16.8, 13.299999999999999, 10.799999999999999],
         }
     )
-
+    ref['City'] = ref['City'].astype("category")
     # init params
```

Comment:
combine with feature reference does not operate over categorical variables. i would not modify this tests.